recipe/meta.yaml

```diff
@@ -1,29 +1,37 @@
-{% set version = "58.2" %}
+{% set version = "60.2" %}
```

Comment:
I believe that only `aarch64` will use this version, unless we update the pin, so I'm OK merging this and we can slowly test it with the rest of the stack before bumping the pin everywhere.